### PR TITLE
[CC][Master] Update choreo connect menu

### PR DESCRIPTION
**Pull request by suksw**
## Purpose
$subject

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: en/mkdocs.yml [hunk 1]
```diff
@@ -228,6 +228,7 @@ nav:
                         - Third Party Key Manager: deploy-and-publish/deploy-on-gateway/choreo-connect/concepts/third-party-key-managers.md
                     - As a Standalone Gateway: deploy-and-publish/deploy-on-gateway/choreo-connect/concepts/as-a-standalone-gateway.md
                     - Subscription Validation: deploy-and-publish/deploy-on-gateway/choreo-connect/concepts/subscription-validation.md
+                    - Configure Multiple Gateways with Choreo Connect: deploy-and-publish/deploy-on-gateway/choreo-connect/configure-multiple-gateways-with-choreo-connect.md
```
Comment (pubudu538) [inline en/mkdocs.yml:231]: We should not list in the Concept menu.

File: en/mkdocs.yml [hunk 2]
```diff
@@ -237,6 +238,28 @@ nav:
                     - Exposing via Custom Hostnames: deploy-and-publish/deploy-on-gateway/choreo-connect/deploy-api/deploy-api-with-custom-hostnames.md
                     - Immutable Gateway:
                         - Deploying as an Immutable Gateway: deploy-and-publish/deploy-on-gateway/choreo-connect/deploy-api/deploy-apis-as-immutable-gateway.md    
+                - Security:
+                    - Authentication:
+                        - API Authentication: deploy-and-publish/deploy-on-gateway/choreo-connect/security/api-authentication/api-authentication.md
+                        - Internal Key Authentication: deploy-and-publish/deploy-on-gateway/choreo-connect/security/api-authentication/internal-key-authentication.md
+                        - API Key Authentication: deploy-and-publish/deploy-on-gateway/choreo-connect/security/api-authentication/api-key-authentication.md
+                        - OAuth2.0 Access Tokens: deploy-and-publish/deploy-on-gateway/choreo-connect/security/api-authentication/oauth2-access-tokens.md
+                        - Test JWT: deploy-and-publish/deploy-on-gateway/choreo-connect/security/generate-a-test-jwt.md
```
Comment (pubudu538) [inline en/mkdocs.yml:247]: Test Token (JWT)

File: en/mkdocs.yml (hunk 1 above)
Comment (suksw) [reply en/mkdocs.yml]: Fixed with https://github.com/wso2/docs-apim/pull/5161/commits/20c93edf8d58d0f82a02a3b4204f3a64954d3550

File: en/mkdocs.yml (hunk 2 above)
Comment (suksw) [reply en/mkdocs.yml]: Fixed with https://github.com/wso2/docs-apim/pull/5161/commits/20c93edf8d58d0f82a02a3b4204f3a64954d3550

**Review by pubudu538 — APPROVED** [review]

**Review by ruthryi — APPROVED** [review]

**Pull request merged**